system-tests/systemTestUtils.mjs

```diff
@@ -37,15 +37,22 @@ export async function caseFilesForFix(caseNumber, ext = 'css') {
 	return tempPath;
 }
 
-export function prepForSnapshot({ results, cwd, output, ...rest }) {
+export function prepForSnapshot({ results, cwd, output, report, ...rest }) {
 	// If output isn't fixed code
 	if (output.startsWith('[')) {
 		// The `source` of each file varies between platforms or if a tmp file is used
-		output = JSON.parse(output).map((warning) => {
-			delete warning.source;
+		output = JSON.parse(output).map((warning) => ({
+			...warning,
```

Comment:
question

For consistency sake, should we refactor l61 and l62 at the same time?

Reply:
Nice catch. Fixed ad05959ae.